### PR TITLE
[PaddleNLP] support Summarization pipeline; upgrade version

### PR DESCRIPTION
**Pull request by sijunhe**
- upgrade version
- support summarization pipeline
- switch all unit test models to `PaddleCI` models

### CODE REVIEW

File: docker_images/common/tests/test_api_summarization.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        self.assertEqual(
+            response.status_code,
+            200,
+        )
+
+        content = json.loads(response.content)
+        self.assertEqual(type(content), list)
+        self.assertEqual(len(content), 1)
+        for result in content:
+            self.assertIn("generated_text", result)
```
Comment (Narsil) [inline docker_images/common/tests/test_api_summarization.py:52]:
```suggestion
            self.assertIn("summary_text", result)
```

File: docker_images/paddlenlp/tests/test_api_summarization.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        self.assertEqual(
+            response.status_code,
+            200,
+        )
+
+        content = json.loads(response.content)
+        self.assertEqual(type(content), list)
+        self.assertEqual(len(content), 1)
+        for result in content:
+            self.assertIn("generated_text", result)
```
Comment (Narsil) [inline docker_images/paddlenlp/tests/test_api_summarization.py:52]:
```suggestion
            self.assertIn("summary_text", result)
```

File: docker_images/paddlenlp/tests/test_api_summarization.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        self.assertEqual(
+            response.status_code,
+            200,
+        )
+
+        content = json.loads(response.content)
+        self.assertEqual(type(content), list)
+        self.assertEqual(len(content), len(text))
+        for result in content:
+            self.assertIn("generated_text", result)
```
Comment (Narsil) [inline docker_images/paddlenlp/tests/test_api_summarization.py:68]:
```suggestion
            self.assertIn("summary_text", result)
```

**Review by Narsil — APPROVED** [review]
LGTM, just modified the output key to keep in line with what was done in `transformers`.

File: docker_images/common/app/pipelines/summarization.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        raise NotImplementedError(
+            "Please implement SummarizationPipeline __init__ function"
+        )
+
+    def __call__(self, inputs: Union[str, List[str]]) -> List[Dict[str, str]]:
+        """
+        Args:
+            inputs (:obj:`str`): a string to be summarized
+        Return:
+            A :obj:`dict`:. The object return should be like {"summary_text": "The string after summarization"}
```
Comment (osanseviero) [inline docker_images/common/app/pipelines/summarization.py:21]: The return object is a list of a dictionary

File: docker_images/common/tests/test_api_summarization.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+            200,
+        )
+
+        content = json.loads(response.content)
+        self.assertEqual(type(content), list)
+        self.assertEqual(len(content), 1)
+        for result in content:
+            self.assertIn("summary_text", result)
+
+    def test_batch_input(self):
```
Comment (osanseviero) [inline docker_images/common/tests/test_api_summarization.py:54]: We don't do batch input in the API afaik

File: docker_images/paddlenlp/app/pipelines/summarization.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+from paddlenlp.taskflow import Taskflow
+
+
+class SummarizationPipeline(Pipeline):
+    def __init__(self, model_id: str):
+        self.taskflow = Taskflow(
+            "text_summarization", task_path=model_id, from_hf_hub=True
+        )
+
+    def __call__(self, inputs: Union[str, List[str]]) -> List[Dict[str, str]]:
```
Comment (osanseviero) [inline docker_images/paddlenlp/app/pipelines/summarization.py:13]: To update to align to common's feedback

**Review by osanseviero — COMMENTED** [review]
My review came very late but I think there were some issues here

**Pull request merged**